core/src/main/java/com/google/zxing/EncodeHintType.java

```diff
@@ -115,4 +115,9 @@
    * {@link String } value).
    */
   GS1_FORMAT,
+
+  /**
+   * Forces which encoding will be used. Introduced for Code-128 code sets (Type {@link String}).
```

Comment:
You could say here that it's currently only used by the Code 128 implementation, and that the possible values are A B and C.

Reply:
I usually tend not to write "currently" in a comment, since the code often changes faster than the comments, but I get your point.

Reply:
The code really won't change much.